### PR TITLE
Add windows sample port

### PR DESCRIPTION
**Pull request by hihigupt**

### CODE REVIEW

**Comment by hihigupt** [comment]
Please ignore files under demos/projects/PC/windows/WinPCap for review.

File: demos/projects/PC/windows/msvc_flags.cmake [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+set(MCU_C_FLAGS "/wd4127" /D_DEBUG
+    /D_CRT_SECURE_NO_WARNINGS
+    /DWIN32 CACHE INTERNAL "MCU build flags")
+add_compile_options(${MCU_C_FLAGS}})
+add_link_options(/NODEFAULTLIB:libcmtd.lib)
```
Comment (danewalton) [inline demos/projects/PC/windows/msvc_flags.cmake:8]: is this to suppress the warning which we get in the embedded SKD? should this be something like borrowed from that?
```
add_compile_options(
      $<$<CONFIG:>:/MT> #---------|
      $<$<CONFIG:Debug>:/MTd> #---|-- Statically link the runtime libraries
      $<$<CONFIG:Release>:/MT> #--|
  )
```

File: demos/projects/PC/windows/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+# set global path
+set(BOARD_DEMO_CONFIG_PATH ${CMAKE_CURRENT_SOURCE_DIR}/config CACHE INTERNAL "Config path")
+set(BOARD_DEMO_FREERTOS_PORT_PATH ${FreeRTOS_Posix_PATH} CACHE INTERNAL "FreeRTOS Port used ")
```
Comment (danewalton) [inline demos/projects/PC/windows/CMakeLists.txt:6]: should this be posix?

File: .github/scripts/ci_tests_windows.ps1 [hunk 3]
```diff
@@ -0,0 +1,49 @@
+#! /bin/bash
```
Comment (danewalton) [inline .github/scripts/ci_tests_windows.ps1:1]: not bin bash

**Review by danewalton — APPROVED** [review]
🚢

File: demos/projects/PC/windows/msvc_flags.cmake (hunk 1 above)
Comment (hihigupt) [reply demos/projects/PC/windows/msvc_flags.cmake:12]: added

**Pull request merged**